brevbaker-api-model-mal/src/main/kotlin/no/nav/pensjon/brev/api/model/Sakstype.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.pensjon.brev.api.model
```

Comment:
Ikke helt enig i navnet og pakke-stien. "Api-model" har vi brukt for samling av dto klasser til nå. Navngivning er alltid vanskelig, men kanskje brevbaker-template, eller noe sånnt.

Reply:
Har litt lyst å ha en type brevbaker-core som har alt som kan gjenbrukes utenfor po-pensjon. Ikke sikkert vi trenger å starte med det, men det hadde vært nice. Så kan vi ha en pensjon-spesifikk en for psak. Ikke at vi bør brukes utenfor pensjon enda(utenom etterlatte). Igjen, ikke sikkert vi må ta det steget nå, men ser det hadde gitt mening.

Reply:
Til det med pakkesti: Har ikkje tatt stilling til det (enno), berre prøvd å halde alt i samme pakke som det var før refaktoriseringa, for å ikkje få massevis av import-endringar her, som blir ekstra støy. Så er einig i at det treng ein ny runde.

Reply:
Jeg tenker at modul-navnet her passer bedre med "brevbaker-dsl", evt. noe annet enn dsl, men ikke api-model.

Reply:
Flere alternativer: "brevbaker-template" eller "brevbaker-mal"